lib/topic_query.rb

```diff
@@ -8,23 +8,41 @@
 class TopicQuery
   include PrivateMessageLists
 
-  PG_MAX_INT ||= 2_147_483_647
-  DEFAULT_PER_PAGE_COUNT ||= 30
```

Comment:
So weird, do you know why it was ||= originally?

Reply:
We were not following autoloading convention in the past which caused files to be required twice. When files are required twice, a constant redefinition warning is triggered by Ruby and our developers used `||=` to avoid it.